### PR TITLE
drop z-stream stable upgrade ci

### PR DESCRIPTION
**Pull request by jianlinliu**
stable upgrade ci is mainly for testing signature related features and some special testing that only can be covered in stable upgrade, per the discussion [conclusion](https://redhat-internal.slack.com/archives/CF8SMALS1/p1724832594292739?thread_ts=1724829611.385049&cid=CF8SMALS1), we will only keep y-stream, chain upgrade stable ci, so drop z-stream stable upgrade ci here.

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @jiajliu @JianLi-RH @shellyyang1989 to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml [hunk 1]
```diff
@@ -1,121 +0,0 @@
...
-      cpu: 100m
-      memory: 200Mi
-tests:
-- as: aws-ipi-disc-priv-tp-f60
-  cron: 24 17 12 2,4,6,8,10,12 *
-  steps:
-    cluster_profile: aws-qe
-    env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
-      ENABLE_OTA_TEST: OCP-69968
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml:71]: @jiajliu can we delete the job with the OTA cases here ?

Comment (jiajliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml:71]: yes, we can

**Comment by jianlinliu** [comment]
cc @liangxia to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml [hunk 2]
```diff
@@ -1,121 +0,0 @@
...
-      FIPS_ENABLED: "true"
-    test:
-    - chain: openshift-upgrade-qe-sanity-disconnected
-    workflow: cucushift-installer-rehearse-gcp-ipi-disconnected-private-cco-manual-workload-identity
-- as: gcp-ipi-proxy-tp-f60
-  cron: 3 19 29 2,4,6,8,10,12 *
-  steps:
-    cluster_profile: gcp-qe
-    env:
-      ENABLE_OTA_TEST: OCP-69948
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml:101]: @jiajliu  and this one

Comment (jiajliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml:101]: it also could be removed

**Pull request merged**